.github/workflows/ci.yml

```diff
@@ -157,6 +159,13 @@ jobs:
       - spack-yaml-checks
       - validate
       - check-versions-exist
+    # The conditional below asserts that we should run this job as long as the result of the dependent jobs
+    # are not 'failures' or 'cancelled' (aka, either 'success' or 'skipped'). Skipped jobs may be created
+    # if certain files are not updated (for example, we don't need to check the validity of json if none
+    # has been updated in the PR), so we should treat 'skipped' jobs as a vacous 'success'.
```

Comment:
Thanks for the explanatory comment. Helps a lot. Is this supposed to be "vacuous 'success'"?

Reply:
Whoops. Spelling!

Reply:
IRK